.github/CODEOWNERS

```diff
@@ -9,25 +9,25 @@
-src/products/vamcSystem/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
-src/products/healthCareLocalFacility/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
-src/products/vamcHealthServicesListing/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
-src/products/locationsListing/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
-src/products/resourcesSupport/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
-src/products/vbaFacility/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
-src/products/questionAnswer/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+# Code ownership of individual components/products
+src/components/event/ @department-of-veterans-affairs/ap-team
+src/components/eventListing/ @department-of-veterans-affairs/ap-team
+src/components/staffProfile/ @department-of-veterans-affairs/ap-team
+src/components/newsStory/ @department-of-veterans-affairs/ap-team
+src/components/storyListing/ @department-of-veterans-affairs/ap-team
+src/components/pressRelease/ @department-of-veterans-affairs/ap-team
+src/components/pressReleaseListing/ @department-of-veterans-affairs/ap-team
+src/components/leadershipListing/ @department-of-veterans-affairs/ap-team
+src/components/vamcSystemVaPolice/ @department-of-veterans-affairs/ap-team
+src/components/vetCenter/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/vetCenterLocationListing/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/vetCenterOutstation/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/vamcSystem/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/healthCareLocalFacility/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/vamcHealthServicesListing/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/locationsListing/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/resourcesSupport/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/vbaFacility/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
+src/components/questionAnswer/ @department-of-veterans-affairs/ap-team @department-of-veterans-affairs/va-platform-cop-frontend
```

Comment:
It feels like this change should have been part of the other PR, but 🤷 they are all going to the same place anyway.

Reply:
https://github.com/department-of-veterans-affairs/next-build/pull/1257#issuecomment-3221151481 😢